Classes/UI/HSGroupViewController.m

```diff
@@ -47,7 +47,7 @@ - (void)viewDidLoad
     [self.kbSource prepareKB:^{
         [self.tableView reloadData];
     } failure:^(NSError* e){
-        UIAlertView* alertView = [[UIAlertView alloc] initWithTitle:@"Couldnt load articles" message:@"Error in loading articles. Please check your internet connection." delegate:nil cancelButtonTitle:@"Ok" otherButtonTitles:nil];
+        UIAlertView* alertView = [[UIAlertView alloc] initWithTitle:@"Couldn't Load Article" message:@"There was an error loading this article. Please check your internet connection." delegate:nil cancelButtonTitle:@"Ok" otherButtonTitles:nil];
```

Comment:
Since we're here anyway, can we change "Ok" to "OK" throughout the HelpStack code? It would make it consistent with the rest of the apps and Apple's usage. Sorry!